flow360/component/flow360_params/flow360_params.py

```diff
@@ -564,7 +596,7 @@ class RampCFL(Flow360BaseModel):
     Ramp CFL for time stepping component
     """
 
-    type: str = pd.Field("ramp", const=True)
+    model_type: Literal["Ramp"] = pd.Field("Ramp", alias="modelType", const=True)
```

Comment:
same here, type is passed to solver, I think we should not change it

Reply:
Reverted to "type"